src/commands/actor/push-data.ts

```diff
@@ -0,0 +1,45 @@
+import { Args } from '@oclif/core';
+
+import { APIFY_STORAGE_TYPES, getApifyStorageClient, getDefaultStorageId } from '../../lib/actor.js';
+import { ApifyCommand } from '../../lib/apify_command.js';
+
+export class PushDataCommand extends ApifyCommand<typeof PushDataCommand> {
+    static override description = 'Stores an object or an array of objects to the default dataset of the actor run.\n'
+    + 'It is possible to pass data using item argument or stdin.\n'
+    + 'Passing data using argument:\n'
+    + '$ apify actor:push-data {"foo": "bar"}\n'
+    + 'Passing data using stdin with pipe:\n'
+    + '$ cat ./test.json | apify actor:push-data\n';
```

Comment:
i guess this should be indented (except the first line)
```suggestion
    static override description = 'Stores an object or an array of objects to the default dataset of the actor run.\n'
        + 'It is possible to pass data using item argument or stdin.\n'
        + 'Passing data using argument:\n'
        + '$ apify actor:push-data {"foo": "bar"}\n'
        + 'Passing data using stdin with pipe:\n'
        + '$ cat ./test.json | apify actor:push-data\n';
```

Reply:
Not sure why it's not indenting, we really need to fix our eslint rules (or rather use some more maintained ones, like sapphire or eslint-config-neon)